src/apps/project-home/AssignmentsView/Wizard/Team/Team.tsx

```diff
@@ -35,127 +38,132 @@ interface TeamProps {
   onBack(): void;
 
   onNext(): void;
-
 }
 
 interface Member {
-
   id: string;
 
   name: string;
 
   user: UserProfile;
 
   since: string;
-
 }
 
 type Field = 'name' | 'since';
 
 interface Sorting {
-
   field: Field;
 
   ascending: boolean;
-
 }
 
 // Flattens list of groups to the list of members, sorted by 'since'
-const getMembers = (groups: Group[], sorting?: Sorting): Member[] => groups
-  .reduce((members, group) => ([
-    ...members, 
-    ...group.members.map(({ user, since }) => 
-      ({ 
-        id: user.id, 
-        name: user.nickname!,  // TODO just a hack for now
-        user, 
-        since 
-      }))
-  ]), [] as Member[])
-  .sort((a, b) => {
-    if (sorting) {
-      const { field } = sorting;
-      const order = sorting.ascending ? -1 : 1;
-      return (a[field] < b[field]) ? -order : (a[field] > b[field]) ? order : 0;
-    } else {
-      return 0;
-    }
-  });
+const getMembers = (groups: Group[], sorting?: Sorting): Member[] =>
+  groups
+    .filter((g) => !g.is_admin)
+    .reduce(
+      (members, group) => [
+        ...members,
+        ...group.members.map(({ user, since }) => ({
+          id: user.id,
+          name: user.nickname!, // TODO just a hack for now
+          user,
+          since,
+        })),
+      ],
+      [] as Member[]
+    )
+    .sort((a, b) => {
+      if (sorting) {
+        const { field } = sorting;
+        const order = sorting.ascending ? -1 : 1;
+        return a[field] < b[field] ? -order : a[field] > b[field] ? order : 0;
+      } else {
+        return 0;
+      }
+    });
 
 export const Team = (props: TeamProps) => {
-
   const { t } = props.i18n;
 
-  const [sorting, setSorting] = useState<Sorting | undefined>({ field: 'name', ascending: false });
+  const [sorting, setSorting] = useState<Sorting | undefined>({
+    field: 'name',
+    ascending: false,
+  });
 
   const members = getMembers(props.project.groups, sorting);
 
-  const { 
-    selected, 
-    toggleSelected, 
-    toggleAll, 
-    isAllSelected 
-  } = useSelectableRows(members, props.assignment.team.map(u => u.id));
+  const { selected, toggleSelected, toggleAll, isAllSelected } =
+    useSelectableRows(
+      members,
+      props.assignment.team.map((u) => u.id)
+    );
 
   useEffect(() => {
-    const members = props.project.groups.reduce((members, group) => (
-      [...members, ...group.members.map(m => m.user).filter(u => selected.includes(u.id))]
-    ), [] as UserProfile[]);
+    const members = props.project.groups.reduce(
+      (members, group) => [
+        ...members,
+        ...group.members
+          .map((m) => m.user)
+          .filter((u) => selected.includes(u.id)),
+      ],
+      [] as UserProfile[]
+    );
 
     props.onChange(members);
   }, [selected, props.project.groups, props.onChange]);
```

Comment:
I noticed that this component drops into an infinite re-render loop. (This is probably an error that I introduced in the first place :-) Anyways: `props.onChange` shouldn't be in the dependency array in this case.